Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -246,5 +206,70 @@ public static string MapRelativeFilePath(MixedRealityToolkitModuleType module, s
             }
             return null;
         }
+
+        private static readonly Dictionary<string, MixedRealityToolkitModuleType> moduleNameMap = new Dictionary<string, MixedRealityToolkitModuleType>()
+        {
+            { "", MixedRealityToolkitModuleType.Core },
```

Comment:
Is it possible to have a unit test that validates that all of the MixedRealityToolkitModuleType values are covered by this dictionary? Otherwise it would be easy to add a new type and forget to add a value here.

Reply:
I've added loops to the unit test which check that each of the module enum values is found.